controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -116,6 +116,13 @@ static std::string parse_bss_info(const std::string &bss_info_str,
         return std::string();
     }
 
+
+    if (confs.size() == 2)
+    {
+        return al_mac;
+    }
```

Comment:
Our convention is:
```cpp
if (cond) {
   // code
}
```
As a general rule, always use clang-format.sh and compile before you commit.
It will save you comments, and fixups.